### ISSUE TITLE
Wi-Fi is not enabled when AG iptables-based protection is enabled

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardForAndroid#1822]
At some point user enables Wi-Fi:
```
23:45:48.089 [single-pool-1-thread-1] INFO  c.a.a.r.NetworkStateReceiver - Start handling android.net.wifi.WIFI_STATE_CHANGED asynchronously
23:45:48.100 [single-pool-1-thread-1] INFO  c.a.a.r.NetworkStateReceiver - Handle Wi-Fi state change
23:45:48.105 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Found IPv6 address for network name:rmnet_data0 (rmnet_data0)
23:45:48.109 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: dummy0 connected = true
23:45:48.110 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data2 connected = false
23:45:48.110 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data3 connected = false
23:45:48.111 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data0 connected = false
23:45:48.111 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: lo connected = true
23:45:48.111 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data1 connected = false
23:45:48.111 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: sit0 connected = false
23:45:48.112 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data7 connected = false
23:45:48.112 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data0 connected = true
23:45:48.112 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: p2p0 connected = true
23:45:48.112 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: wlan0 connected = true
23:45:48.113 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data2 connected = true
23:45:48.113 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data1 connected = true
23:45:48.113 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data4 connected = false
23:45:48.113 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data3 connected = false
23:45:48.114 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data6 connected = false
23:45:48.114 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_data5 connected = false
23:45:48.114 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data8 connected = false
23:45:48.114 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data6 connected = false
23:45:48.115 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data7 connected = false
23:45:48.115 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data4 connected = false
23:45:48.115 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: r_rmnet_data5 connected = false
23:45:48.115 [single-pool-1-thread-1] INFO  c.a.android.filtering.commons.d - Interface: rmnet_ipa0 connected = true
23:45:48.116 [single-pool-1-thread-1] INFO  c.a.android.filtering.filter.e - handleWiFiChanged: Network hash=1010jionet, oldState=2, newState=3
```

**Expected:** Shortly after that we should receive a CONNECTIVITY_CHANGE event. 
**Actual:** However, it does not happen unless user disables AdGuard manually.

### Your environment

* Adguard version: 2.11.76
* Adguard filtering mode: Proxy (auto)
* Device model: Xiaomi Redmi Note 4
* Operating system and version: 8.1

**Comment by ameshkov** [comment]
I suppose the cause of it is the way we resolved #1640

**Comment by ameshkov** [comment]
@sfionov I've confirmed that the issue is caused by the `AdGUARD_INPUT` table.

However, what causes it is one of the "restoring" commands.
